### PR TITLE
resolves CLI warnings

### PR DESCRIPTION
**Pull request by danielpeintner**

### CODE REVIEW

File: packages/cli/src/cli.ts [hunk 1]
```diff
@@ -205,6 +205,7 @@ const args = program.args;
 
 // .env parsing
 const env: dotenv.DotenvConfigOutput = dotenv.config();
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 if (env.error && (env.error as any).code && (env.error as any).code !== "ENOENT") {
```
Comment (relu91) [inline packages/cli/src/cli.ts:209]: Can't we cast to `Error`?

**Review by relu91 — CHANGES_REQUESTED** [review]

File: packages/cli/src/cli.ts (hunk 1 above)
Comment (danielpeintner) [reply packages/cli/src/cli.ts:209]: Unfortunately not...

`error TS2339: Property 'code' does not exist on type 'Error'`

I also tried more dedicated errors extending `Error` interface... but wasn't successful to find one.. not sure which kind does have `code`.
Do you?

Comment (relu91) [reply packages/cli/src/cli.ts:209]: https://stackoverflow.com/a/49562477/925955 what about this answer? is it pretty old and I'm quite puzzled that it wasn't fixed... but still worth trying?

Comment (danielpeintner) [reply packages/cli/src/cli.ts]: Good finding. That works!
I updated the PR accordingly...

**Review by relu91 — APPROVED** [review]

**Pull request merged**